rollup.config.js

```diff
@@ -29,13 +32,23 @@ const config = mode => ({
         if (source === 'ReactDOM') {
           return {id: 'react-dom', external: true};
         }
+        if (source === 'ReactNative') {
+          return {id: 'react-native', external: true};
+        }
         return null;
       },
     },
-    nodeResolve(),
+    nodeResolve({
+      extensions: target === 'native' ? ['.native.js', '.js'] : undefined,
```

Comment:
This seems error prone, if a `.native.js` file is missing or has a typo, the browser version would be resolved instead. Was there an issue you discovered with the rollup config proposed in #108?